src/functionalTest/java/uk/gov/hmcts/sptribs/testutil/CcdCaseCreator.java

```diff
@@ -75,4 +75,50 @@ public String createInitialStartEventAndSubmit(String eventId,
 
         return caseDetails.getId().toString();
     }
+
+    public void createInitialStartEventAndSubmitSystemEvent(String eventId,
```

Comment:
Is it better to generalise the creation method than overload for different user tokens?

Reply:
I think it would but my concern was with so many ongoing functional test tickets it could cause issues with conflicts.